freqtrade/freqtradebot.py

```diff
@@ -237,16 +237,25 @@ def _refresh_whitelist(self, whitelist: List[str]) -> List[str]:
 
         return final_list
 
-    def get_target_bid(self, ticker: Dict[str, float]) -> float:
+    def get_target_bid(self, pair: str) -> float:
         """
         Calculates bid target between current ask price and last price
         :param ticker: Ticker to use for getting Ask and Last Price
         :return: float: Price
         """
-        if ticker['ask'] < ticker['last']:
-            return ticker['ask']
-        balance = self.config['bid_strategy']['ask_last_balance']
-        return ticker['ask'] + balance * (ticker['last'] - ticker['ask'])
+
+        if self.config['bid_strategy']['use_book_order']:
```

Comment:
This will is dangerous with old configs (and causes one of the test failures ...)

better use ` if self.config['bid_strategy'].get(['use_book_order'], False):` ...